base_app/src/app/app-tab-overlay/app-tab-overlay.component.ts

```diff
@@ -464,23 +464,21 @@ export class AppTabOverlayComponent implements AfterViewInit, OnInit {
             console.log('newStyle: ' + newStyle);
             console.log('newValueStyle: ' + newValueStyle);
 
-            const retVal = this.sendCssUpdate('update-css', cssDef, cssValueDef);
-            console.log(retVal);
+            this.sendCssUpdate('update-css', cssDef, cssValueDef);
         }
 
         console.log(cssDef);
     }
 
-    onDragOver(ev: DragEvent) {
+    onDragOver(ev: DragEvent): void {
```

Comment:
This function can be deleted if we've remove drag n drop.

Reply:
Function has been deleted